search/search_repository_blackbox_test.go

```diff
@@ -58,102 +61,213 @@ func (s *searchRepositoryBlackboxTest) getTestFixture() *tf.TestFixture {
 			case 0:
 				wi.Type = fxt.WorkItemTypes[1].ID
 				wi.Fields[workitem.SystemTitle] = "Test TestRestrictByType"
-				wi.Fields[workitem.SystemState] = "closed"
 			case 1:
 				wi.Type = fxt.WorkItemTypes[2].ID
 				wi.Fields[workitem.SystemTitle] = "Test TestRestrictByType 2"
-				wi.Fields[workitem.SystemState] = "closed"
 			}
 			return nil
 		}),
-	)
+	}
+	allRecipes := append(baseRecipes, extraRecipeFuncs...)
+	return tf.NewTestFixture(s.T(), s.DB, allRecipes...)
```

Comment:
Actually this will crash horribly (panic) when a test from below fails because this line will fail the outmost test case (in testify speech) and not the individual one. I suggest to add a required `t *testing.T` parameter to the `getTestFixture` function.